api/nodes/serializers.py

```diff
@@ -313,6 +314,13 @@ def update(self, instance, validated_data):
         pass
 
 
+class NodeLinksDetailSerializer(NodeLinksSerializer):
+    """
+    Overrides NodeLinksSerializer to make id required.
+    """
+    id = IDField(source='_id', required=True)
```

Comment:
For updating/deleting, the `id` is required to be in the body of the request per json-api v1. (Updating requires `id` for both single and bulk, deleting just needs `id` for bulk).  But I don't want the `id` to be required when I'm creating a resource - it doesn't exist yet, hence the two versions of the serializers, one requiring id, one not.

However, since we don't update node_links, and DRF doesn't validate for delete, we probably don't need it here. But I still need it for the other serializers.